### ISSUE TITLE
[Feature Request] Support for REI

### ISSUE THREAD

**Issue by vizthex123** [ticticboooomer/MasterfulMachinery#69]
[REI](https://www.curseforge.com/minecraft/mc-mods/roughly-enough-items) is a much better version of JEI, and it'd be nice to have full support for it since I've had a few minor issues with Masterful Machinery not rendering things properly.

**Comment by ticticboooomer** [comment]
I agree it would be nice, but given there are plenty of other more important issues waiting for dev. this is not going to be anywhere near high priority

**Comment by vizthex123** [comment]
yeah, that's fine. Just figured i'd post it in case other people had the same question.
